probe/endpoint/ebpf.go

```diff
@@ -0,0 +1,238 @@
+package endpoint
+
+import (
+	"bytes"
+	"encoding/binary"
+	"net"
+	"sync"
+	"unsafe"
+
+	log "github.com/Sirupsen/logrus"
+	bpflib "github.com/kinvolk/gobpf-elf-loader/bpf"
+)
+
+import "C"
+
+var byteOrder binary.ByteOrder
+
+type EventType uint32
+
+const (
+	_ EventType = iota
+	EventConnect
+	EventAccept
+	EventClose
```

Comment:
I would add a comment saying the constant should be in sync with the equivalent definitions in the ebpf program.